Documentation/CRDs/Object-Storage/ceph-object-multisite-crd.md

```diff
@@ -111,3 +112,5 @@ The pools allow all of the settings defined in the Pool CRD spec. For more detai
  In the case of multiple stores (or multiple endpoints for a single store), you are not required to put all endpoints in this list. Only specify the endpoints that should be used for replication traffic.
 
  If you update `customEndpoints` to return to an empty list, you must the Rook operator to automatically add the CephObjectStore service endpoint to Ceph's internal configuration.
+
+ * `preservePoolsOnDelete`: If it is set to 'true' the pools used to support the zone will remain when the ceph object zone CR will be deleted. This is a security measure to avoid accidental loss of data. It is set to 'true' by default. If not specified is also deemed as 'true'.
```

Comment:
```suggestion

 * `preservePoolsOnDelete`: If it is set to 'true' the pools used to support the CephObjectZone will remain when it is deleted. This is a security measure to avoid accidental loss of data. It is set to 'true' by default. If not specified is also deemed as 'true'.
```